roles/v2/pool/pool-config.toml

```diff
@@ -1,6 +1,6 @@
 listen_address = "0.0.0.0:34254"
-# tp_address = "127.0.0.1:8442"
-tp_address = "75.119.150.111:8442"
```

Comment:
Let's keep the pool pointing at the hosted TP still.

Reply:
wrong deletion, as the others